scripts/release.ts

```diff
@@ -66,6 +67,64 @@ async function undoLastCommit() {
   console.log(c.green('Publish commit deleted.'));
 }
 
+const newDesignSystemVersion = (): string => {
+  const newDesignSystemVersionJSON = JSON.parse(sh(`npm version -w @cmsgov/design-system --json`));
+  return newDesignSystemVersionJSON['@cmsgov/design-system'];
+};
+
+const updateDSVersion = (json: JSON | any): JSON => {
+  const dsKey = '@cmsgov/design-system';
+  json.dependencies[dsKey] = newDesignSystemVersion();
+  return json;
+};
+
+/*
+ * Utilize `npm ls` to get all members of our design system workspace in JSON format.
+ * This allows us to dynamically update our child package.json files later on
+ *  rather than explicitly enumerating our examples and child design systems.
+ * The info we want (member name and package.json file location) are at different levels in the JSON,
+ *  so we have to dig a bit.
+ * We grab the package names by getting all of the keys in the object associated with the 'dependencies' key.
+ * We loop over the names to get the package.json file locations that are in the object associated with the 'resolved' key.
+ */
+const getPackageNamesAndLocations = (): Array<Array<string>> => {
+  const packageJson: any = JSON.parse(sh('npm ls -ws @cmsgov/design-system --json'));
+  const deps = packageJson['dependencies'];
+  const packageNames = Object.keys(deps);
+  return packageNames.map((name: any) => {
+    return [name, deps[name]['resolved']];
```

Comment:
what do you think about returning an object here?
```
{
  packageName: string,
  packageJsonFileLocation: string
}
```
I think that would improve the readability for destructuring in the `updatePackageJson` function.

Plus, `packageJsonFileLocation` could be based on `path.join(root, cleanedPackageLocation, 'package.json')` if you move the package location cleaup logic into this function